### PR TITLE
Added delaymon for printing values of NtDelayExecution(...)

### PR DESCRIPTION
**Pull request by nystilaps**
Added delaymon for printing values of NtDelayExecution(...) function calls

### CODE REVIEW

**Comment by nystilaps** [comment]
Good point! Thank you.
It didn't occur to me until plugin was finished.

**Comment by nystilaps** [comment]
Changed to kernel version.

File: src/plugins/delaymon/delaymon.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#include "delaymon.h"
+
+#include <cmath>
+
+#define PLUGIN_NAME "[DELAYMON]"
+
+// Works on Windows 7
```
Comment (tklengyel) [inline src/plugins/delaymon/delaymon.cpp]: Does this mean this only works for Win7 or that it only has been tested with Win7? If it's the latter, please make sure the plugin bails on startup when ran against the wrong version.

Comment (nystilaps) [reply src/plugins/delaymon/delaymon.cpp]: This is copy-pasted comment is wrong.

Comment (nystilaps) [reply src/plugins/delaymon/delaymon.cpp]: It must work everywhere

Comment (tklengyel) [reply src/plugins/delaymon/delaymon.cpp]: If the comment is wrong then just remove it please

Comment (nystilaps) [reply src/plugins/delaymon/delaymon.cpp]: Removed

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
Thanks!

**Pull request merged**